defaults/aws_config.yaml

```diff
@@ -8,7 +8,7 @@ instance_type_loader: 'c6i.xlarge'
 instance_type_monitor: 't3.large'
 # manual on creating loader AMI's: see docs/new_loader_ami.md
 ami_id_loader: 'scylla-qa-loader-ami-v19'
-ami_id_monitor: 'ubuntu/images/hvm-ssd/ubuntu-jammy-22.04-amd64-server-20231207' # ubuntu/images/hvm-ssd/ubuntu-jammy-22.04-amd64-server-20231207Canonical, Ubuntu, 22.04 LTS, amd64 jammy image build on 2023-12-07
+ami_id_monitor: 'scylladb-monitor-4-6-2-2024-02-13t08-06-04z'
```

Comment:
worth to add link/hint to procedure how and when to update this image.
E.g. I see we copy it to multiple regions (change in utils/copy_ami_to_all_regions.sh)

Reply:
added a new doc for it, and linking to it from here